src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="annonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img src="images/imac.jpeg"
+      alt="computer__imac__27__monoblock_retina_5k_apple_a1419"
+      class="card__img"
+      >
+
+      <p class="card__p1">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__p2">
+        Product code: 195434
+      </p>
+
+      <div class="card__starbox">
+        <div class="card__stars">
+          <div class="card__star modifier"></div>
+          <div class="card__star modifier"></div>
+          <div class="card__star modifier"></div>
+          <div class="card__star modifier"></div>
+          <div class="card__star"></div>
+        </div>
+        <div class="card__reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="card__price-div">
+        <span class="card__price-1">
+          Price:
+        </span>
+        <span class="card__price-2">
+          $2,199
+        </span>
```

Comment:
```suggestion
        <span class="card__price-label">
          Price:
        </span>
        <span class="card__price-amount">
          $2,199
        </span>
```
please use it like that - it's way more descriptive